doc/specs/stdlib_math.md

```diff
@@ -90,3 +90,142 @@ program demo_clip_real
   ! clipped_value <- 3.02500010
 end program demo_clip_real
 ```
+
+### `linspace` - Create a linearly spaced rank one array
+
+#### Description
+
+Returns a linearly spaced rank 1 array from [`start`, `end`]. Optionally, you can specify the length of the returned array by passing `n`.
+
+#### Syntax
+
+`res = [[stdlib_math(module):linspace(interface)]] (start, end [, n])`
+
+#### Status
+
+Experimental
+
+#### Class
+
+function.
+
+#### Argument(s)
+
+`start`: Shall be scalar of any numeric type. This argument is `intent(in)`.
+`end`: Shall be the same `type` and `kind` as `start`. This argument is `intent(in)`.
+`n`: Shall be an integer specifying the length of the output. This argument is `intent(in)`.
+
+#### Output value or Result value
+
+The output is a rank 1 array of `type` and `kind`, whose length is either 100 (default value) or `n`.
+
+If `n` == 1, return a rank 1 array whose only element is `end`.
+If `n` <= 0, return a rank 1 array with length 0
+
```

Comment:
It should include what will happen with integer arguments.

Reply:
Included in latest snapshot